### PR TITLE
Run this repo's pre-commit hooks in tox (and thus travis)

### PR DESCRIPTION
**Pull request by chriskuehl**
seems like a good idea. not sure if there's some edge case where this doesn't make sense (or where it needs to be a separate target)?

### CODE REVIEW

**Comment by chriskuehl** [comment]
retest this please

**Comment by chriskuehl** [comment]
Fails because argparse isn't part of Python 2.6:
```
diff --git a/pre_commit/clientlib/validate_base.py b/pre_commit/clientlib/validate_base.py
index 707bdde..af18725 100644
--- a/pre_commit/clientlib/validate_base.py
+++ b/pre_commit/clientlib/validate_base.py
@@ -1,11 +1,11 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
-import argparse
 import os.path
 import re
 import sys
 
+import argparse
 import jsonschema
 import jsonschema.exceptions
 import yaml
diff --git a/pre_commit/main.py b/pre_commit/main.py
index aed6688..1f52c33 100644
--- a/pre_commit/main.py
+++ b/pre_commit/main.py
@@ -1,9 +1,9 @@
 from __future__ import unicode_literals
 
-import argparse
 import os
 import sys
 
+import argparse
 import pkg_resources
 
 from pre_commit import color
diff --git a/tests/main_test.py b/tests/main_test.py
index 140b587..ac1674f 100644
--- a/tests/main_test.py
+++ b/tests/main_test.py
@@ -2,7 +2,6 @@ from __future__ import absolute_import
 from __future__ import unicode_literals
 
 import argparse
-
 import mock
 import pytest
```

Probably need to just run this on Python 2.7, I'll follow up to this later. Spent some time perusing @asottile's aspy.refactor_imports, but looks like it would be somewhat complicated to add a way to fake classify 3rd-party modules as part of the standard library for old Python versions.

**Comment by asottile** [comment]
@chriskuehl if you put `language_version: python2.7` on reorder imports I think this is merge able

File: pre_commit/clientlib/validate_base.py [hunk 1]
```diff
@@ -3,12 +3,12 @@
 
 import argparse
 import os.path
-import pkg_resources
```
Comment (chriskuehl) [inline pre_commit/clientlib/validate_base.py]: heh, this was definitely my fault. at least i recognized that "someone" would make a dumb mistake like this!

**Comment by chriskuehl** [comment]
fyi the commit you merged was fine even though it failed appveyor. I think I angered it by pushing too frequently.

**Comment by asottile** [comment]
Also apparently "you cannot merge a request with failing checks from a phone". Switch to desktop version works though.  Dumb feature github :/

**Pull request merged**